lightly/api/api_workflow_compute_worker.py

```diff
@@ -88,13 +91,18 @@ def schedule_compute_worker_run(
             lightly_config:
                 Lightly configuration. All possible values are listed in our
                 docs: https://docs.lightly.ai/lightly.cli.html#default-settings
+            selection_config:
```

Comment:
Please also add the docstring to `create_compute_worker_config()`

Reply:
Thanks 👍